console/console-init/ui/src/Routes.tsx

```diff
@@ -52,7 +52,7 @@ export const AppRoutes = () => (
   //   {/* iot-project detail page route */}
   //   <LazyRoute
   //     path="/iot-projects/:namespace/:projectname/:sublist"
-  //     getComponent={getIoTProjectDetailPage}
+  //     getComponent={getProjectDetailPage}
```

Comment:
I will suggest name as IoTProjectDetail as it will create confusion in messaging and iot projects